pkg/kubelet/network/dns/dns.go

```diff
@@ -105,16 +110,7 @@ func (c *Configurer) getClusterDNS(pod *v1.Pod) ([]net.IP, error) {
 		klog.Errorf("failed to get network %s/%s: %v", pod.Tenant, networkName, err)
 	}
 
-	if utilfeature.DefaultFeatureGate.Enabled(features.MandatoryArktosNetwork) {
-		return nil, fmt.Errorf("network %s/%s not found", pod.Tenant, networkName)
-	}
-
-	// fallback to default cluster DNS for pods on the default network
```

Comment:
nit: suggest to keep this default network behavior (though it does not matter when the feature gate is disabled by default).

Reply:
updated per comment.